src/main/java/network/brightspots/rcv/HartCvrReader.java

```diff
@@ -125,9 +124,6 @@ void readCastVoteRecord(List<CastVoteRecord> castVoteRecords, Path path) throws
           Logger.info("Parsed %d cast vote records.", castVoteRecords.size());
         }
       }
-    } catch (Exception exception) {
-      Logger.severe("Error parsing cast vote record:\n%s", exception);
-      throw exception;
     }
```

Comment:
Did this error logging get moved elsewhere?

Reply:
Oops, incorrect removal. Restored.

Reply:
Actually, I take that back: this is from SpotBugs:

> M D REC: Exception is caught when Exception is not thrown in network.brightspots.rcv.HartCvrReader.readCastVoteRecord(List, Path)  At HartCvrReader.java:[line 127]

Reply:
Weird... if an `Exception` isn't thrown, why do we need to `try` at all? I'm a little suspicious of that Spotbugs warning.

Reply:
I think it's complaining that the thing thrown is IOException, rather than Exception. Restored with that change and it's happy now.